website/content/en/preview/AWS/user-data.md

```diff
@@ -11,6 +11,16 @@ This document describes how you can customize the UserData and AMIs for your EC2
 ## Configuration
 
 In order to specify custom user data and AMIs, you must include them within a AWSNodeTemplate resource. You can then reference this AWSNodeTemplate resource through `spec.providerRef` in your provisioner.
+```yaml
```

Comment:
Should we also surface this information at the Provisioner API and getting started sections of our docs so that it's more visible to users?